LoopBack/LBPersistedModel.h

```diff
@@ -7,12 +7,13 @@
 
 #import "LBModel.h"
 
+
+/**
+ * Blocks of this type are executed for a successful method invocation that returns
+ * an LBPersistedModel instance as a callback argument.
+ */
 @class LBPersistedModel;
 typedef void (^LBPersistedModelObjectSuccessBlock)(LBPersistedModel *model);
-typedef void (^LBPersistedModelArraySuccessBlock)(NSArray *array);
-typedef void (^LBPersistedModelVoidSuccessBlock)();
-typedef void (^LBPersistedModelBoolSuccessBlock)(BOOL boolean);
-typedef void (^LBPersistedModelNumberSuccessBlock)(NSInteger number);
```

Comment:
Hmm, isn't this a breaking change? I mean if there is any existing user code relying on `LBPersistedModelVoidSuccessBlock` (for example), then it will break after upgrading to a new version of iOS SDK.

Can we keep these typedefs around, perhaps with a deprecation notice (if there is a mechanism for that).

```io
typedef LBModelVoidSuccessBlock LBPersistedModelVoidSuccessBlock;
```